ui/components/src/Legend/Legend.stories.tsx

```diff
@@ -45,16 +45,25 @@ function generateMockLegendData(count: number): LegendProps['data'] {
 // Simple wrapper to try to help visualize that the legend is positioned absolutely
 // inside a relative ancestor.
 const LegendWrapper = (props: LegendProps) => {
+  const {
+    options: { position },
+  } = props;
+
   return (
     <Box
       sx={{
-        position: 'relative',
-        width: props.width + 100,
-        height: props.height + 100,
         border: (theme) => `solid 1px ${theme.palette.divider}`,
```

Comment:
Separated out the box with the border and the box with the height/width to avoid some issues with being off by a few pixels  for the legend because of a combo of the border and `box-sizing: border-box`.